fastlane/lib/fastlane/actions/upload_symbols_to_crashlytics.rb

```diff
@@ -133,6 +159,17 @@ def self.available_options
                                        verify_block: proc do |value|
                                          available = ['ios', 'appletvos', 'mac']
                                          UI.user_error!("Invalid platform '#{value}', must be #{available.join(', ')}") unless available.include?(value)
+                                       end),
+          FastlaneCore::ConfigItem.new(key: :dsym_worker_threads,
+                                       env_name: "FL_UPLOAD_SYMBOLS_TO_CRASHLYTICS_DSYM_WORKER_THREADS",
+                                       default_value: '1',
```

Comment:
Shouldn't this be `1` as a number, instead of `'1'` as a string?

Reply:
My first little Ruby ❤️. Fixed.